src/main/java/com/flightstats/hub/config/properties/AwsProperties.java

```diff
@@ -35,4 +35,6 @@ public String getSigningRegion() {
         return propertiesLoader.getProperty("aws.signing_region", "us-east-1");
     }
 
+    public String getDisasterRecoveryRegion() { return propertiesLoader.getProperty("aws.disaster.recovery.region", "us-west-2"); }
```

Comment:
Just verifying that this does the right thing if there's no DR bucket. Looks like it does, since you're checking for bucket name in a couple places. I think staging/dev/integration tests all lack DR buckets?

Reply:
yeah, even if it gets called when there is no corollary bucket it will just return the default